### ISSUE TITLE
About loss function

### ISSUE THREAD

**Issue by lkyee** [zjuela/LapSRN-tensorflow#12]
Hi，
thanks for your nice code! I want to ask a question.

###========================== DEFINE TRAIN OPS ==========================###
    loss2   = compute_charbonnier_loss(net_image2.outputs, t_target_image, is_mean=True)
    loss1   = compute_charbonnier_loss(net_image1.outputs, t_target_image_down, is_mean=True)
    g_loss  = loss1 + loss2 * 4

I don't understand why loss2 should multiply 4.
Look forward your kind reply!
Thank you!
